docs/images/DELETEME

```diff
@@ -0,0 +1 @@
+Images for the guide go here.
```

Comment:
What is this?

Reply:
Hi Matt I have removed this file now. Thanks.